### PR TITLE
Pycascades 2018

### LINKED ISSUE #455: PyCascades 2018
22 - 23 January, 2018

~~~ yaml
- title: PyCascades 2018
  dir: pycascades-2018
  youtube_lists: 
    - https://www.youtube.com/playlist?list=PLcNrB7gPa-NeTAsgtUx48EXCVN63brZe3
    - https://www.youtube.com/playlist?list=PLcNrB7gPa-NcF_-QQpI5uHnvJGnQkHRUQ
  language: eng
  related_urls:
  - label: event schedule
    url: https://www.pycascades.com/schedule/
~~~

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Fixes #455

### CODE REVIEW

**Comment by Daniel-at-github** [comment]
Seems that @mikeckennedy will talk of this videos in the next podcast in https://pythonbytes.fm/
Publishing it before gives they a good excuse to talk about pyvideo.

**Comment by Daniel-at-github** [comment]
Videos unpublished (yet):
* Guido van Rossum: BDFL Python 3 retrospective
* Anna Liao: RaspberryPy to RustyPi: Porting a Python module to Rust
* Anna Schneider: Navigating unconscious bias
* Łukasz Langa: Gradual Typing of Production Applications

**Comment by mikeckennedy** [comment]
Thanks for the mention @Daniel-at-github I covered Emily's AST talk on [episode 152 of talk python](https://talkpython.fm/152) and Lukas's talk on gradual typing on [talk python 151](https://talkpython.fm/151). :)

**Comment by Mariatta** [comment]
Thanks for working on adding PyCascades videos to PyVideo :)

Anna Liao was not recorded (her own request), so you can tick that off :)

**Comment by Daniel-at-github** [comment]
Thanks for the tip @Mariatta!

Any data else that you want to contribute (from this, from video channels that we miss, ...) please let as know (via comment, issue or pull request) :innocent:

File: pycascades-2018/videos/anna-schneider-navigating-unconscious-bias.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {
+      "label": "event schedule",
+      "url": "https://www.pycascades.com/schedule/"
+    },
+    {
+      "label": "talk schedule",
+      "url": "https://www.pycascades.com/talks/navigating-unconscious-bias/"
+    },
+    {
+      "label": "event schedule",
```
Comment (Mariatta) [inline pycascades-2018/videos/anna-schneider-navigating-unconscious-bias.json]: I think the label here should be "slides".

**Comment by Mariatta** [comment]
Thanks @Daniel-at-github :) Everything looks good. Only noticed one link was improperly labeled.
I think you got everything else covered.
Thanks again!

**Comment by Daniel-at-github** [comment]
Some url have changed, have to make time to take a look.

File: pycascades-2018/videos/brett-cannon-setting-expectations-for-open-source-participation.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "tags": [
+    "community"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/COULfomh-Nk/maxresdefault.jpg",
+  "title": "Setting Expectations for Open Source Participation",
+  "videos": [
+    {
+      "type": "youtube",
+      "url": "https://www.youtube.com/watch?v=COULfomh-Nk"
```
Comment (Mariatta) [inline pycascades-2018/videos/brett-cannon-setting-expectations-for-open-source-participation.json]: https://www.youtube.com/watch?v=-Nk-8fSJM6I

Comment (Daniel-at-github) [reply pycascades-2018/videos/brett-cannon-setting-expectations-for-open-source-participation.json]: The URL in [this lines](https://github.com/Daniel-at-github/data/blob/7e892b8d8160ed5f1663d9e1548db33429be489b/pycascades-2018/videos/dhruv-govil-python-for-feature-film.json#L22-28)
~~~ json
  "title": "Python for Feature Film",
  "videos": [
    {
      "type": "youtube",
      "url": "https://www.youtube.com/watch?v=s6i2FG0Y1oA"
    }
  ]
~~~
Is now https://www.youtube.com/watch?v=qGhKvEA2aNo

Comment (Daniel-at-github) [reply pycascades-2018/videos/brett-cannon-setting-expectations-for-open-source-participation.json]: ... and some more.
Hope it's OK now.

**Comment by Mariatta** [comment]
Hi, I've added a new commit to this PR. I've added all the "talk schedule" links. Hope that's ok :)
Let me know what else I can help with.

**Comment by Daniel-at-github** [comment]
Thanks for the commit! :smile:
You can help telling more people to contribute here:
* Anyone can add an issue of a missing video channel of a conference, or add more data to an existing video (tags, slides, ...)
* Easy project to learn json, github dinamics and view python conference videos.
* A way to contribute back to the Python ecosystem.

**Comment by redapple** [comment]
I just noticed that 3 talks have a recorded date in February 2018, while the conference was on Jan 22 & 23 2018.
"BDFL Python 3 retrospective", Wed 21 February 2018, By Guido van Rossum
"Gradual Typing of Production Applications", Wed 21 February 2018, By Łukasz Langa
"Navigating unconscious bias", Wed 21 February 2018, By Anna Schneider

**Comment by Daniel-at-github** [comment]
@redapple, set it to Jan 22 in #487

**Pull request merged**